CONTRIBUTING.md

```diff
@@ -98,3 +112,15 @@ The website listens on port 8081.
 
 The best place to get help is to come [join the discord](https://discord.gg/FHpfstT7Dw). There is
 a "contributors" channel dedicated to helping contributors.
+
+## Common errors
```

Comment:
@Ashvith10 you also mentioned:

```shell
Error: EEXIST: file already exists, mkdir 'path/to/tybalt/node_modules/aria-build/node_modules/esbuild/.install'
```

But I missed the resolution. Do you recall? Is it worth including here?

Reply:
@doug-wade looks like some side-effect of using Nix flakes. I was able to resolve this issue by using `git clean -xfd`.

Reply:
Added!